tasks/kubectl-apply-ddl-patches.yml

```diff
@@ -0,0 +1,42 @@
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: eu.gcr.io/census-gcr/gcloud-kubectl
+
+params:
+  GCP_PROJECT_NAME:
+  KUBERNETES_CLUSTER:
+  KUBERNETES_FILE_PATH:
+  SERVICE_ACCOUNT_JSON:
+  WAIT_UNTIL_AVAILABLE_TIMEOUT:
+
+inputs:
+  - name: kubernetes-repo
+run:
+  path: sh
+  args:
+    - -exc
+    - |
+      cat >~/gcloud-service-key.json <<EOL
+      $SERVICE_ACCOUNT_JSON
+      EOL
+
+      # Use gcloud service account to configure kubectl
+      gcloud auth activate-service-account --key-file ~/gcloud-service-key.json
+      gcloud container clusters get-credentials ${KUBERNETES_CLUSTER} --zone europe-west2 --project ${GCP_PROJECT_NAME}
+
+      kubectl delete deploy census-rm-ddl --force --grace-period=0 --wait || true
+      sleep 10
```

Comment:
Not ideal but we had a lot of problems with the `kubectl delete` not behaving itself, it would exit the command and take minutes to delete the pod despite forcing it or telling the command to wait for deletion.

Reply:
Wouldn't surprise me if it needed a higher sleep. For terminating the QID-Batch-Runner in the AT theres a sleep for like 60s

Reply:
We've done away with this now in favour of deleting the pod (with a wait) straight after the deployment is deleted. Seems to work okay!